app/models.py

```diff
@@ -22,8 +16,61 @@ class User(Base):
     password = Column(String, nullable=False)
     first_name = Column(String, nullable=False)
     last_name = Column(String, nullable=False)
-    gender = Column(Enum("male", "female",), nullable=False)
+    gender = Column(Enum("male", "female", name="gender_enum"), nullable=False)
     age = Column(Integer, nullable=False)
     height = Column(Integer, nullable=False)
     weight = Column(Float, nullable=False)
+    activity_level = Column(Integer, nullable=False)
     user_created_at = Column(TIMESTAMP(timezone=True), nullable=False, server_default=text("NOW()"))
+
+class Tdee(Base):
+    __tablename__ = "tdees"
+    user_id = Column(Integer,ForeignKey("users.user_id", ondelete="CASCADE"), primary_key=True, nullable=False)
+    tdee_number = Column(Integer, nullable=False)
+    tdee_created_at = Column(TIMESTAMP(timezone=True), nullable=False,  primary_key=True, server_default=text("NOW()"))
+
+class Bmi(Base):
+    __tablename__ = "bmis"
+    user_id = Column(Integer,ForeignKey("users.user_id", ondelete="CASCADE"), primary_key=True, nullable=False)
+    bmi_number = Column(Integer, nullable=False)
+    bmi_created_at = Column(TIMESTAMP(timezone=True), nullable=False,  primary_key=True, server_default=text("NOW()"))
+
+
+class Food(Base):
+    __tablename__ = "foods"
+    food_name = Column(String, primary_key=True, nullable=False)
+    food_calories = Column(Integer, nullable=False)
+    food_created_at = Column(TIMESTAMP(timezone=True), nullable=False, server_default=text("NOW()"))
+
+
+class CaloriesConsumptions(Base):
+    __tablename__ = "calories_consumptions"
+    user_id = Column(Integer,ForeignKey("users.user_id", ondelete="CASCADE"), primary_key=True, nullable=False)
+    eaten_food_name = Column(String, nullable=False)
```

Comment:
It seems that you may have duplicated data in this table and in table Food. I guess the neater solution would be storing all info about food (calories, nutrition value, etc) in table Food and then in table CaloriesConsumption store only food_id.